extensions/ql-vscode/src/skeleton-query-wizard.ts

```diff
@@ -98,6 +103,38 @@ export class SkeletonQueryWizard {
     });
   }
 
+  public async determineStoragePath() {
+    const firstStorageFolder = getFirstWorkspaceFolder();
+
+    if (isCodespacesTemplate()) {
+      return firstStorageFolder;
+    }
+
+    let storageFolder = getSkeletonWizardFolder();
+
+    if (storageFolder === undefined || !existsSync(storageFolder)) {
+      storageFolder = await Window.showInputBox({
+        title:
+          "Please choose a folder in which to create your new query pack. You can change this in the extension settings.",
+        value: firstStorageFolder,
+        ignoreFocusOut: true,
+      });
+    }
+
+    if (storageFolder === undefined) {
+      throw new UserCancellationException("No storage folder entered.");
+    }
+
+    if (!existsSync(storageFolder)) {
+      throw new UserCancellationException(
+        "Invalid folder. Must be a folder that already exists.",
+      );
+    }
+
+    await setSkeletonWizardFolder(storageFolder);
```

Comment:
This didn't work for me, and I got the following error:
```
Unable to write to User Settings because codeQL.skeletonWizard.folder is not a registered configuration. (codeQL.createQuery)
```

Adding the config option to package.json might fix this!